.github/workflows/tests.yml

```diff
@@ -13,11 +13,11 @@ jobs:
       - uses: actions/checkout@v3
       - uses: software-mansion/setup-scarb@v1
         with:
-          scarb-version: "2.6.3"
+          scarb-version: "2.7.0"
```

Comment:
Suggestion - let's use a `.tool-versions` file. The CI actions should pick it up and since it's shared in the repo, everyone working on it will be using the same toolchain.